app/src/androidTest/java/ch/epfl/sdp/blindly/main_screen/ProfilePageTest.kt

```diff
@@ -143,29 +144,6 @@ class ProfilePageTest {
 
     }
 
-    //TODO not yet implemented
-    /*@Test
-    fun recordButtonFiresRecordingActivity() {
-        //Create and show the audio player
-        onView(withId(R.id.play_audio_profile_button)).perform(click())
-
-        onView(withId(R.id.record_button)).perform(click())
-        intended(hasComponent(RecordingActivity::class.java.name))
-    }
-
-    @Test
-    fun recordButtonRemovedAudioPlayerFragment() {
-        //Create and show the audio player
-        onView(withId(R.id.play_audio_profile_button)).perform(click())
-
-        onView(withId(R.id.record_button)).perform(click())
-        val fragmentManager = fragment.childFragmentManager
-        val audioFragment = fragmentManager.findFragmentById(R.id.fragment_audio_container_view)
-        assert(audioFragment == null)
-    }
-
-     */
-
     private fun getAudioPlayerFragment(): Fragment? {
```

Comment:
Why is this private function hanging out here all alone :D ? It seems other tests are deleted and no one is using this function.